### PR TITLE
logging for sshd DEBUG -> INFO

### PR DESCRIPTION
**Pull request by robin13**

### CODE REVIEW

**Comment by pvizeli** [comment]
What kind of information are around on log file after that change?

**Comment by robin13** [comment]
INFO level.  This is the default level for openssh.  Shows connections and disconnections and much more.

e.g.
```
Sep  9 15:40:31 my-websrv sshd[460]: Received signal 15; terminating.
Sep  9 15:40:32 my-websrv sshd[5767]: Server listening on 0.0.0.0 port 22.
Sep  9 15:40:32 my-websrv sshd[5767]: Server listening on :: port 22.
Sep  9 15:40:49 my-websrv sshd[2541]: Received disconnect from 1.2.3.4 port 32460:11: disconnected by user
Sep  9 15:40:49 my-websrv sshd[2541]: Disconnected from user foo 4.3.2.1 port 32460
Sep  9 15:40:50 my-websrv sshd[5954]: Accepted publickey for foo from 4.3.2.1 port 32401 ssh2: RSA SHA256:efe473564cb63a7bf025dd691ef0ae0ac906c03ab408375b9094e326c2ad9a76
```

File: ssh/config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "SSH server",
-  "version": "6.3",
+  "version": "6.3.1",
```
Comment (pvizeli) [inline ssh/config.json]: -> 6.4

Comment (robin13) [reply ssh/config.json]: Done.

File: ssh/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.3.1
```
Comment (pvizeli) [inline ssh/CHANGELOG.md]: wrong version

Comment (robin13) [reply ssh/CHANGELOG.md]: :man_facepalming: fixed.

**Comment by bwolther** [comment]
Looks good. No more debug output to console at login (i.e. no more "debug1: permanently_set_uid: 0/0").

**Pull request merged**